src/Events.php

```diff
@@ -64,6 +65,7 @@ public function init() {
 	 */
 	public function do_wc_admin_daily() {
 		WC_Admin_Notes_New_Sales_Record::possibly_add_sales_record_note();
+		WC_Admin_Notes_Giving_Feedback_Notes::add_notes_for_admin_giving_feedback();
```

Comment:
Side note: if we merge in we'll need to make sure this gets updated to `WC_Admin_Notes_Giving_Feedback_Notes::possibly_add_note();`

Reply:
I'm not actually sure how this got here, it's duplicated from below. I've removed this line.